src/redux/state.js

```diff
@@ -52,13 +55,22 @@ const store = {
     this._state.ProfilePages.textArea = '';
     this._renderFunc(this._state);
   },
-  updateArea(NewText) {
+  _updateArea(NewText) {
     this._state.ProfilePages.textArea = NewText;
     this._renderFunc(this._state);
   },
-  subscribe(observer) {
-    this._renderFunc = observer;
+  dispatch(action) {
+    switch (action.type) {
+      case 'ADD-POST':
```

Comment:
Consider creating a file named "actionTypes.js" and storing there an object with all the action names.
`const ACTION_TYPES = { ..., ADD_POST: 'ADD_POST', ...};`
This way you can import this object everywhere you need and reference needed type like this
`switch(ACTION_TYPES.ADD_POST)`

Reply:
I added new file and I used it in state and mypost